homeassistant/components/tplink/light.py

```diff
@@ -210,7 +218,7 @@ def _async_extract_brightness_transition(
         if (brightness := kwargs.get(ATTR_BRIGHTNESS)) is not None:
             brightness = round((brightness * 100.0) / 255.0)
 
-        if self.device.is_dimmer and transition is None:
+        if self.device.device_type == DeviceType.Dimmer and transition is None:
```

Comment:
This might actually be fixed in the library now

Reply:
If not, we sould definitely fix it for 0.7.0.

Reply:
Fix in https://github.com/python-kasa/python-kasa/pull/929